ingeniamotion/fsoe.py

```diff
@@ -0,0 +1,266 @@
+from typing import TYPE_CHECKING, Dict
+
+import ingenialogger
+from fsoe_master.fsoe_master import Dictionary, DictionaryItem, MasterHandler, Watchdog
+from ingenialink.pdo import RPDOMap, RPDOMapItem, TPDOMap, TPDOMapItem
+
+from ingeniamotion.metaclass import DEFAULT_SERVO
+
+if TYPE_CHECKING:
+    from ingeniamotion.motion_controller import MotionController
+
+
+class FSoEMasterHandler:
+    """FSoE Master Handler.
+
+    Args:
+        slave_address: The servo's FSoE address.
+        connection_id: The FSoE connection ID.
+        watchdog_timeout: The FSoE master watchdog timeout in seconds.
+
+    """
+
+    def __init__(self, slave_address: int, connection_id: int, watchdog_timeout: float):
+        self.__master_handler = MasterHandler(
+            dictionary=self._saco_phase_1_dictionary(),
+            slave_address=slave_address,
+            connection_id=connection_id,
+            watchdog_timeout_s=watchdog_timeout,
+            application_parameters=[],
```

Comment:
I think the only application parameter that will be transmitted (and should be added here) is the SRA.

On slave this is hard coded: 
https://bitbucket.org/novantamotion/fsoe_master_python/src/3fa5d29e4a25da9c1fb9406059fa20b0f41e4a57/virtual_slave/fsoe_slave/state/fsoe_state_connection.c#lines-101

There's a helper function to generate the SRA here: `https://bitbucket.org/novantamotion/fsoe_master_python/pull-requests/31/diff`

Reply:
It will be done on a different issue.

Reply:
Create it and place it for next sprint, so we don't forget during planning